apps/synapse-portal-framework/src/components/SectionLayout.tsx

```diff
@@ -39,9 +39,11 @@ export function SectionLayout(props: SectionLayoutProps) {
 
   return (
     <Container
-      maxWidth={'lg'}
-      {...ContainerProps}
-      sx={{ overflow: 'auto', ...ContainerProps?.sx }}
+      maxWidth={ContainerProps?.maxWidth ?? 'lg'}
+      sx={{
+        overflow: 'auto',
+        ...ContainerProps?.sx,
+      }}
```

Comment:
Revert this, you're currently dropping all other ContainerProps
```suggestion
      maxWidth={'lg'}
      {...ContainerProps}
      sx={{ overflow: 'auto', ...ContainerProps?.sx }}
```